data/hurd/gnu.png.license

```diff
@@ -1,5 +1,6 @@
 Copyright 2003 Free Software Foundation, Inc,
 Copyright 2003 Aurelio A. Heckert <aurium@gmail.com>,
 Copyright 2010 Paolo Caroni <kenren89@gmail.com>
+Copyright 2022 Joseph Engelhardt
```

Comment:
please add email

Reply:
did you change those files ? if not i am not sure your copyright  should be listed here

Reply:
I did change gnu.* and hurd.* to combine elements of the gray and red tables.

Reply:
ok so it's a derivate work since you're not author of the main I think only the only creator should be preserved, what do you think ?

Reply:
I'm okay with that if you are.

Reply:
ok thx can you please squash commits to make it clearer and I'll test it this weekend